modules/document_repository/php/document_repository.class.inc

```diff
@@ -79,9 +79,9 @@ class Document_Repository extends \NDB_Menu_Filter
                           'v.Date_uploaded',
                          );
 
-        $openAccordion = $_GET['openAccordion'];
+        $openAccordion = $_GET['openAccordion'] ?? '';
         $this->tpl_data['openAccordion'] = $openAccordion;
-        $filtered = $_GET['filtered'];
+        $filtered = isset($_GET['filtered']) ? true : false;
```

Comment:
This should be switched to `? false : true;` or else the file categories don't appear which changes the default behaviour of the module.

Reply:
maybe it should just be `$filtered = !empty($_GET['filtered'])`?